e2e/test/scenarios/onboarding/onboarding-checklist.cy.spec.ts

```diff
@@ -27,6 +26,44 @@ describe("Onboarding checklist page", () => {
       ).should("not.exist");
     });
   });
+
+  it("the link should be dismissible from the sidebar", () => {
+    cy.get("[data-accordion=true]")
+      .as("accordionElement")
+      .findByRole("heading", { name: "Start visualizing your data" })
+      .should("be.visible");
+
+    cy.findByTestId("main-navbar-root")
+      .findByRole("listitem", { name: "How to use Metabase" })
+      .as("sidebarOnboardingLink")
+      .realHover()
+      .icon("eye_crossed_out")
+      .as("dismissIcon")
+      .should("be.visible")
+      .trigger("mouseenter");
```

Comment:
It just reduces the flakiness A LOT.
I don't think we can (or that we should) abstract it into a helper because it is highly specific to this case where you (1) need to hover the parent in order to reveal an icon, and then (2) hover the icon itself in order to see a tooltip 🫠

Reply:
A comment explaining that would help someone like me here

(suggestion, not a blocker)

Reply:
https://github.com/metabase/metabase/pull/50536/commits/25976186728ac0317d0bb13108049d9df66212be